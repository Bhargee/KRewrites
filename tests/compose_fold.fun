fold (fun a b -> a + b) map (fun c -> c + 1)
