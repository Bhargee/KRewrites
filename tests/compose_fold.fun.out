<T>
    <k>
        fold ((( fun ((a b) -> (a + b)) )) -| (( fun (c -> (c + 1)) )))
          
    </k>
    <store>
        .Map
    </store>
    <env>
        .Map
    </env>
</T>
